### PR TITLE
[2.13.x] DDF-4848 Change java installation doc duplication

### PR DESCRIPTION
**Pull request by austinsteffes**
#### What does this PR do?
Fixes documentation to not have duplicated sections. Improves the sections wording to ensure installation goes smoothly.
#### Who is reviewing it? 

@ahoffer @tyler30clemens 
#### Select relevant component teams: 
@codice/docs 

#### Ask 2 committers to review/merge the PR and tag them here.

@brjeter
@ricklarsen - Documentation

#### How should this be tested?

successful quickbuild and check documentation
#### What are the relevant tickets?

For GH Issues:
Fixes: #4848 

#### Checklist:
- [X] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by austinsteffes** [comment]
build now

**Comment by austinsteffes** [comment]
build now

**Comment by austinsteffes** [comment]
build now

File: distribution/docs/src/main/resources/content/_installing/java-reqs.adoc [hunk 1]
```diff
@@ -22,8 +22,13 @@ For a development system:
...
+[NOTE]
+====
+Prior to installing ${branding}, ensure the system time is accurate to prevent federation issues.
+====
+
 .*NIX Unlinking JAVA_HOME if Previously Set
-[WARNING]
+[NOTE]
 ====
 Unlink `JAVA_HOME` if it is already linked to a previous version of the JRE:
```
Comment (ahoffer) [inline distribution/docs/src/main/resources/content/_installing/java-reqs.adoc]: I know this didn't change, but I'm not sure what it means to unlink `JAVA_HOME` home. When I read unlink,I think "use `unlink` to remove a symbolic link. Is that what this means? If so, which link? Or does it mean to use `unset` to unset the shell variable named `JAVA_HOME`? I think the documentation should be clear on this.

File: distribution/docs/src/main/resources/content/_installing/java-reqs.adoc [hunk 2]
```diff
@@ -70,24 +75,20 @@ setx JAVA_HOME "<PATH_TO_JAVA><JAVA_VERSION>"
...
 ----
 +
-. Restart Terminal (shell) or Command Prompt.
-
-* Verify that the `JAVA_HOME` was set correctly.
-====
-
-====
-
+. Restart or open up a new Terminal (shell) or Command Prompt to verify `JAVA_HOME` was set correctly. System restart is not needed for the changes to take effect.
```
Comment (ahoffer) [inline distribution/docs/src/main/resources/content/_installing/java-reqs.adoc]: How about "It is not necessary to restart the system to for the changes to take effect".

File: distribution/docs/src/main/resources/content/_installing/java-reqs.adoc [hunk 3]
```diff
@@ -128,9 +129,17 @@ setx JRE_HOME "<PATH_TO_JAVA><JAVA_VERSION>"
 setx PATH "%PATH%;%JRE_HOME%\bin"
 ----
 +
-. Restart Terminal (shell) or Command Prompt.
-
-* Verify that the `JRE_HOME` was set correctly.
+. Restart or open up a new Terminal (shell) or Command Prompt to verify `JRE_HOME` was set correctly. System restart is not needed for the changes to take effect.
```
Comment (ahoffer) [inline distribution/docs/src/main/resources/content/_installing/java-reqs.adoc]: How about "It is not necessary to restart the system to for the changes to take effect".

File: distribution/docs/src/main/resources/content/_quickstart/quickstart-installing.adoc [hunk 4]
```diff
@@ -22,125 +22,8 @@ For security reasons, ${branding} cannot be started from a user's home directory
...
-echo %JAVA_HOME%
-----
-
-.*nix
-----
-echo $JAVA_HOME
-----
-====
-
-.File Descriptor Limit on Linux
```
Comment (ahoffer) [inline distribution/docs/src/main/resources/content/_quickstart/quickstart-installing.adoc]: :exclamation: The file descriptor limit on linux is not part of the Java path. I don't think this section should be deleted.

File: distribution/docs/src/main/resources/content/_installing/java-reqs.adoc (hunk 1 above)
Comment (austinsteffes) [reply distribution/docs/src/main/resources/content/_installing/java-reqs.adoc]: right below this it says:

`unlink JAVA_HOME`

**Comment by austinsteffes** [comment]
build now

File: distribution/docs/src/main/resources/content/_quickstart/quickstart-installing.adoc (hunk 4 above)
Comment (austinsteffes) [reply distribution/docs/src/main/resources/content/_quickstart/quickstart-installing.adoc]: added back in.

File: distribution/docs/src/main/resources/content/_installing/java-reqs.adoc (hunk 3 above)
Comment (austinsteffes) [reply distribution/docs/src/main/resources/content/_installing/java-reqs.adoc]: changed

File: distribution/docs/src/main/resources/content/_installing/java-reqs.adoc (hunk 2 above)
Comment (austinsteffes) [reply distribution/docs/src/main/resources/content/_installing/java-reqs.adoc]: changed

File: distribution/docs/src/main/resources/content/_installing/java-reqs.adoc (hunk 1 above)
Comment (ahoffer) [reply distribution/docs/src/main/resources/content/_installing/java-reqs.adoc]: I can't think why someone would create a symbolic link named JAVA_HOME in the first place. JAVA_HOME should be the name of shell var.

Comment (austinsteffes) [reply distribution/docs/src/main/resources/content/_installing/java-reqs.adoc]: changing to unset

**Comment by tyler30clemens** [comment]
Hero Successful ✅

**Pull request merged**